### PR TITLE
Fix claim outputs

### PR DESCRIPTION
**Pull request by samuel-rufi**
Fixes https://github.com/iotaledger/wallet.rs/issues/1117

### CODE REVIEW

File: src/account/operations/output_claiming.rs [hunk 1]
```diff
@@ -225,93 +230,82 @@ impl AccountHandle {
...
+                            if let Some(native_tokens) = output_data.output.native_tokens() {
+                                new_native_tokens.add_native_tokens(native_tokens.clone())?;
+                                // Re-calculate the required storage deposit amount since we have added a new native
+                                // token
+                                required_storage_deposit = minimum_storage_deposit(
+                                    &byte_cost_config,
+                                    &first_account_address.address.inner,
+                                    &option_native_token,
+                                )?;
+                            }
```
Comment (Thoralf-M) [inline src/account/operations/output_claiming.rs]: Hm we do that in 288 already each time before we compare, so I guess this is not needed

File: src/account/operations/output_claiming.rs [hunk 2]
```diff
@@ -225,93 +230,82 @@ impl AccountHandle {
...
-            let mut additional_inputs = Vec::new();
 
+            let mut additional_inputs = Vec::new();
             if new_amount < required_storage_deposit {
                 // add more inputs
                 for output_data in &possible_additional_inputs {
                     // Recalculate every time, because new intputs can also add more native tokens, which would increase
                     // the storage deposit cost
-                    let required_storage_deposit = minimum_storage_deposit(
+                    let mut required_storage_deposit = minimum_storage_deposit(
```
Comment (Thoralf-M) [inline src/account/operations/output_claiming.rs]: But maybe here it should be like
```suggestion
                    required_storage_deposit = minimum_storage_deposit(
```
and the first declaration should be mutable

Comment (samuel-rufi) [reply src/account/operations/output_claiming.rs]: fixed

File: src/account/operations/output_claiming.rs (hunk 1 above)
Comment (samuel-rufi) [reply src/account/operations/output_claiming.rs]: fixed

**Review by Thoralf-M — DISMISSED** [review]

**Review by Thoralf-M — APPROVED** [review]

**Review by thibault-martinez — APPROVED** [review]

**Pull request merged**